patches/ui-webui-resources-tools-optimize_webui.gni.patch

```diff
@@ -10,11 +10,11 @@ index 697a4526bbe325d72890d9cdecdde5e01f32b042..1344c74bcee1ec89f031823a65064ca6
  
      external_paths = []
      if (is_chromeos_ash) {
-@@ -86,6 +87,7 @@ template("optimize_webui") {
+@@ -83,6 +84,7 @@ template("optimize_webui") {
+     external_paths += [ "//resources/polymer/v3_0/|$polymer_path" ]
+ 
+     if (!is_chrome_untrusted) {
++      external_paths += [ "chrome://resources/brave/|$brave_resources_path", "//resources/brave/|$brave_resources_path" ]
```

Comment:
see above, but at the very least this should also collapse the patch into one line

Reply:
Condensed into one line here db0ffa4edcb6bc439bba91451d03dbef422bacd8. `optimize_webui` is being replaced by `bundle_js` soon, so I think the duplication will naturally go away shortly (let me know if that sounds OK).